corehq/apps/app_manager/dbaccessors.py

```diff
@@ -100,6 +100,12 @@ def get_latest_build_id(domain, app_id):
     return res['id'] if res else None
 
 
+def get_latest_build_version(domain, app_id):
+    """Get id of the latest build of the application, regardless of star."""
+    res = _get_latest_build_view(domain, app_id, include_docs=False)
+    return res['value']['version'] if res else None
```

Comment:
Mentioned this offline, but this `value` is not the whole application doc. It is only the small dict here: https://github.com/dimagi/commcare-hq/blob/75d3d314658c930f0d5da5a4a5da2c6d041fc237/corehq/apps/app_manager/_design/views/applications_brief/map.js#L3-L17